config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
 end
```

Comment:
- [ **OPTIONAL** ] Good job implementing the required routes of this project. However, I think it would be better if you also add a default route as right now your `root` rout has a blank page. I can suggest this example below.
```ruby
....
root "users#index"

 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
end
```